src/sentry/models/rule.py

```diff
@@ -130,3 +131,18 @@ class RuleActivity(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_ruleactivity"
+
+
+def in_fourteen_days():
+    return timezone.now() + timedelta(days=14)
+
+
+@region_silo_only_model
+class NeglectedRules(Model):
+    __relocation_scope__ = RelocationScope.Organization
+
+    rule_id = FlexibleForeignKey("sentry.Rule")
```

Comment:
@ceorourke I think we should add the `org_id` as well to make it easier to query the organization